src/submap.h

```diff
@@ -418,6 +418,35 @@ class maptile_impl
             return *std::prev( sm->get_items( pos() ).cend() );
         }
 
+        /** Returns the uppermost favorite item if there is a favorite item. Otherwise, returns the item with the greatest volume.
+        *
+        * Assumes there is at least one item.
+        * Used to decide which item to render.
+        * TODO: might be slow enough to justify storing the most visible item for each tile? 
```

Comment:
You can check (and should!) check this yourself.
Spawn a ton of items and see if the game is still playable before and after your changes.

Reply:
I actually did test it ingame, spawned a bunch of items, I mentioned that in the PR but didn't update the comment in the code, mb